ios/src/TangramMap/TGMapViewController.h

```diff
@@ -0,0 +1,731 @@
+ */
+- (BOOL)markerSetPointEased:(TGMapMarkerId)identifier coordinates:(TGGeoPoint)coordinate duration:(float)duration easeType:(TGEaseType)ease;
+
+/**
+ Sets a marker styled to be a polyline (described in a `TGGeoPolyline`).
+
+ @param identifier the marker identifier created with `-markerAdd`
+ @oaram polyline the polyline geometry to use for this marker
+ @return `YES` if this operation was successful, `NO` otherwise
+
+ @note Markers can have their geometry set multiple time wwith possibly different geometry types.
+ */
+- (BOOL)markerSetPolyline:(TGMapMarkerId)identifier polyline:(TGGeoPolyline *)polyline;
+
+/**
+ Sets a marker to be a polygon geometry (described in a `TGGeoPolygon`).
+
+ @param identifier the marker identifier created with `-[TGMapViewController markerAdd]`
+ @param polygon the polygon geometry to use for this marker
+ @return `YES` if this operation was successful, `NO` otherwise
+
+ @note Markers can have their geometry set multiple time with possibly different geometry types.
+ */
+- (BOOL)markerSetPolygon:(TGMapMarkerId)identifier polygon:(TGGeoPolygon *)polygon;
+
+/**
+ Adjusts marker visibility
+
+ @param identifier the marker identifier created with `- markerAdd`
+ @param whether the marker is set to visible
+ @return `YES` if this operation was successful, `NO` otherwise
+ */
+- (BOOL)markerSetVisible:(TGMapMarkerId)identifier visible:(BOOL)visible;
+
+/**
+ Sets an image loaded with a <a href="https://developer.apple.com/reference/uikit/uiimage">
+ UIImage</a> to a marker styled with a <a href="https://mapzen.com/documentation/tangram/Styles-Overview/#points">
+ points style</a>.
+
+ ```obj
+ TGMapMarkerId mid = [vc markerAdd];
+ [mapView markerSetStyling:mid styling:@"{ style: 'points', color: 'white', size: [25px, 25px], order:500 }"];
+ [mapView markerSetPoint:mid coordinates:coordinate];
+ [mapView markerSetImage:mid image:[UIImage imageNamed:@"marker-image.png"]];
+ ```
+
+ @param identifier the marker identifier created with `-markerAdd`
+ @param the `UIImage` that will be used to be displayed on the marker
+ @return `YES` if this operation was successful, `NO` otherwise
+
+ @note An image marker must be styled with a
+ <a href="https://mapzen.com/documentation/tangram/Styles-Overview/#points">point style</a>.
+ */
+- (BOOL)markerSetImage:(TGMapMarkerId)identifier image:(UIImage *)image;
+
+/**
+ Removes a marker for a marker identifier.
+
+ @param identifier the marker identifier created with `-markerAdd`
+ @return `YES` if removal was succesful, `NO` otherwise
+ */
+- (BOOL)markerRemove:(TGMapMarkerId)identifier;
+
+#pragma mark Scene loading - updates interface
+
+/**
+ Loads a scene file synchronously.
+
+ If the scene file is set as a resource in your application bundle, make sure to resolve
+ the path for this URL relative to your bundle (for example `Resources/scene.yaml`).
+
+ If your scene is hosted remotely (any path starting with `https://` is considered a remote scene file)
```

Comment:
[Yes we do](https://github.com/tangrams/tangram-es/blob/master/core/src/util/url.cpp#L391_L392), I'll update the comment here.